src/utility/getIconByType.js

```diff
@@ -0,0 +1,14 @@
+// lodash
+import _lowerCase from 'lodash/lowerCase'
+
+// icons
+import vegIcon from '../assets/images/vegSymbol.svg'
+import nonVegIcon from '../assets/images/nonVeg.svg'
+
+function getIconByType(type){
+    const veg = _lowerCase(type)==="veg"
```

Comment:
don't compare it directly, what if tomorrow in backend instead of **veg** it get changes to **vegetarian**, you can use startsWith